client/src/docs-ui/links-banner/link-banner.tsx

```diff
@@ -21,11 +21,11 @@ export class DocsLinkBanner {
           Amplify GitHub
         </amplify-external-link>
         <amplify-external-link
-          href="https://gitter.im/AWS-Amplify/Lobby"
+          href="https://discord.gg/jWVbPfC"
```

Comment:
Let's move this link to a common place. Maybe we need a common external links file.